example/crawl_async_example.dart

```diff
@@ -39,25 +39,29 @@ Future<AnalysisContext> get analysisContext async {
       throw StateError('Expected to find exactly one context root, got $roots');
     }
 
-    _analysisContext = ContextBuilder().createContext(contextRoot: roots[0]);
+    final analysisContext =
+        ContextBuilder().createContext(contextRoot: roots[0]);
+    _analysisContext = analysisContext;
+    return analysisContext;
   }
 
-  return _analysisContext;
+  return null;
```

Comment:
Why does this return null now? It looks like it should be:
```suggestion
  return _analysisContext!;
```

Reply:
done